ui/src/app/applications/components/application-sync-options.tsx

```diff
@@ -33,10 +33,46 @@ function booleanOption(name: string, label: string, defaultVal: boolean, props:
 
 const optionStyle: React.CSSProperties = {marginTop: '0.5em'};
 
+interface SyncOption {
+    name?: string;
+    key: SyncOptionKey;
+    value?: boolean;
+    default?: boolean;
+}
+
+const enum SyncOptions {
+    Validate = 'Validate',
```

Comment:
This option actually disables validation. Can you rename it to `Skip Schema Validation` and keep it disabled by default?